src/source_files.jl

```diff
@@ -70,17 +70,17 @@ second form, get the line number at the given `byte_index` within `source`.
 source_line(x) = source_line(sourcefile(x), first_byte(x))
 
 """
-    souce_location(x)
-    souce_location(source::SourceFile, byte_index::Integer)
+    source_location(x)
+    source_location(source::SourceFile, byte_index::Integer)
 
-    souce_location(LineNumberNode, x)
-    souce_location(LineNumberNode, source, byte_index)
+    source_location(LineNumberNode, x)
+    source_location(LineNumberNode, source, byte_index)
```

Comment:
haha I'm glad this is only misspelled in the docs :sweat_smile:

Reply:
had a sigh of relief myself noting those misspellings were only in  the docs